dev-db/percona-xtrabackup/percona-xtrabackup-2.4.13.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="Hot backup utility for MySQL based servers"
+HOMEPAGE="https://www.percona.com/software/mysql-database/percona-xtrabackup"
+SRC_URI="https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-${PV}/source/tarball/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	app-arch/lz4:0=
+	app-editors/vim-core
+	<dev-libs/boost-1.65.0:=
+	dev-libs/libaio
+	<dev-libs/libedit-20170329.3.1
```

Comment:
Same here.

Reply:
Since 2.4.10 it seems like it's compatible with newer libedit (https://github.com/percona/percona-xtrabackup/commit/76296545b11db55613e57237c09fe6cc117933f9), so removing this limitation.